api/src/service/domain/workflow/subproject_eventsourcing.ts

```diff
@@ -31,9 +32,9 @@ export function sourceSubprojects(
     if (!event.type.startsWith("subproject_")) {
       continue;
     }
-
     const result = applySubprojectEvent(ctx, subprojects, event);
     if (Result.isErr(result)) {
+      logger.warn(result);
```

Comment:
please remove - we don't want eventsourcing to produce any logs, that's the reason it returns the errors instead